iroha-cpp/services/Query.cpp

```diff
@@ -0,0 +1,163 @@
+#include "GrpcClient.hpp"
+#include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
+#include "GrpcResponseHandler.hpp"
+#include "services/Query.hpp"
+#include "model/converters/pb_common.hpp"
+
+
+namespace IROHA_CPP
+{
+
+Query::Query(const std::string& account_name, uint64_t query_counter, const iroha::keypair_t& keypair)
```

Comment:
IMO it is better to change order:
```suggestion
Query::Query(const std::string& account_name, const iroha::keypair_t& keypair, uint64_t query_counter)
```

It is first time I see `query_counter` - so rather is not that important, maybe default value would be useful

Reply:
I put 0u as a default value.